scenario/createsheet.go

```diff
@@ -66,6 +67,32 @@ func (settings CreateSheetSettings) Execute(sessionState *session.State,
 			return sessionState.IDMap.Add(settings.ID, genObj.GenericId, sessionState.LogEntry)
 		}
 
+		prop, err := genObj.GetPropertiesRaw(ctx)
+		if err != nil {
+			return err
+		}
+
+		// client updates two parameters, do this safely jsonparser as using a struct might "lose" values
+		_, err = jsonparser.GetString(prop, "gridResolution")
+		if err == jsonparser.KeyPathNotFoundError {
+			prop, err = jsonparser.Set(prop, []byte(`"small"`), "gridResolution")
+			if err != nil {
+				return err
+			}
+		}
```

Comment:
Should we return errors that are not `jsonparser.KeyPathNotFoundError`?

Reply:
hmm yes, good point

Reply:
fixed